### DISCUSSION TITLE
Fail to install on new Mac - bash script error

### DISCUSSION THREAD

**Discussion by dheater** [Homebrew/discussions#8] [Everyday usage]
Failing to install `brew` on a new Mac (standard terminal) using the latest install instructions:
```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"

...


HEAD is now at 22d908c85 Merge pull request #8737 from SeekingMeaning/bump-formula-pr/audit-online
/bin/bash: line 111: 11285 Terminated: 15          "$@"
Failed during: /usr/local/bin/brew update --force
```

**Comment by dheater** [comment]
I've tried an uninstall using `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall.sh)"` and deleting everything in `/usr/local/` - no luck

**Comment by jonchang** [comment]
> /usr/local/bin/brew update --force

This likely means that there was a network issue while you were installing. Try running the install script again (it should pick up where it left off for the most part).

**Reply by dheater** [reply]
Error persisted over multiple retries. I haven't seen any other signs of network issues, but everything seems to have installed. It's working so going to leave it alone.

**Comment by QbinWU** [comment]
`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"`

but failed to stay
showing

curl: (7) Failed to connect to raw.githubusercontent.com port 443: Connection refused

**Reply by gromgit** [reply]
Try downloading `https://raw.githubusercontent.com/Homebrew/install/master/install.sh` via your browser, then `/bin/bash -c /path/to/install.sh`.
